doc/source/conf.py

```diff
@@ -319,6 +319,7 @@
         'lsim2 is deprecated',  # Deprecation of scipy.signal.lsim2
         'impulse2 is deprecated',  # Deprecation of scipy.signal.impulse2
         'step2 is deprecated',  # Deprecation of scipy.signal.step2
+        'interp2d` is deprecated',  # Deprecation of scipy.interpolate.interp2d
```

Comment:
That backtick after `interp2d` shouldn't be there.

Reply:
```suggestion
        'interp2d is deprecated',  # Deprecation of scipy.interpolate.interp2d
```

Reply:
Well, that backtick is in the message of the deprecation warning. However, something really strange is going on also with the rendering in the GH UI - the first line is basically invisible:

![interp2_dep_I](https://github.com/scipy/scipy/assets/33685575/f1eb20b0-92a4-4e2a-8704-9e2d9fec22d9)
except when highlighted
![interp2_dep_II](https://github.com/scipy/scipy/assets/33685575/3fbe2931-7aa8-4da9-b71e-9a067cb7b53d)

Reply:
I think it might be easiest to remove the backtick from `both` the deprecation message and the warning filter.

Reply:
Ah, the backticks are actually inserted by `@np.deprecate` 🤦

Reply:
Github rendering aside, the backtick belongs in that error message, so I'm resolving this.